bin/npm2rpm.js

```diff
@@ -49,6 +49,7 @@ tar_extract['stream'].on('finish', () => {
 		license: package_json['license'],
 		description: package_json['description'],
 		project_url: package_json['homepage'],
+		bugs_url: package_json['bugs'].url,
```

Comment:
This line needs to check that `package_json['bugs']` exists,  otherwise it fails with this error (try creating a spec for webpack)

```javascript
/home/daniel/workspace/npm2rpm/bin/npm2rpm.js:52
                bugs_url: package_json['bugs'].url,
                                              ^

TypeError: Cannot read property 'url' of undefined
```